crates/bevy_gltf/src/loader/mod.rs

```diff
@@ -840,11 +787,11 @@ async fn load_gltf<'a, 'b, 'c>(
                         &skin,
                         joints,
                         skinned_mesh_inverse_bindposes[skin.index()].clone(),
-                        get_gltf_extras(skin.extras()),
+                        as_gltf_extras(skin.extras()),
                     );
 
                     let handle = load_context
-                        .add_labeled_asset(skin_label(&skin), gltf_skin)
+                        .add_labeled_asset(skin_label(&skin).to_string(), gltf_skin)
```

Comment:
I think this can use GltfSkin's method? See related comment on `gltf_ext/skin.rs`.

```suggestion
                        .add_labeled_asset(skin.asset_label(&skin).to_string(), gltf_skin)
```